### ISSUE TITLE
What is the supervision of point cloud? 

### ISSUE THREAD

**Issue by czy341181** [chenyilun95/DSGN#14]
What is the supervision of point cloud? 
I have two ways of understanding

1. Is  it the supervision of depth map? It's just that the depth map is generated by the point cloud？
2. Is point cloud used to supervise volume directly

**Comment by czy341181** [comment]
One way is to supervise the voxel occupancy of 3D
grid by ground-truth point cloud using binary cross-entropy
loss. 
The second is to supervise depth on the plane-sweep
cost volume as explained in Section

But I don't see you in the code using raw point cloud supervision， I only see you use the generated depth map

**Comment by chenyilun95** [comment]
Yes. you'll find the loss_disp here https://github.com/Jia-Research-Lab/DSGN/blob/master/tools/train_net.py#L286

**Comment by czy341181** [comment]
I know this, but  this is supervised by the generated depth map. 

Your paper mentioned a point that "One way is to supervise the voxel occupancy of 3D
grid by ground-truth point cloud using binary cross-entropy
loss"  .

Is this not used in the code? Because the effect is not better than another?

**Comment by chenyilun95** [comment]
Yes it is not in this code. My implementation shows it gets wrose result. Its implementation is a bit tricky since only the sparse ray is visiable when supervising the voxel grid. You need to project back the depth map to point cloud and ignore the voxels that are not within the projection ray.

**Comment by czy341181** [comment]
Thanks for your reply! I see.

So in your paper :  "Supervision of point cloud is important"
Here "Supervision of point cloud" represents loss_disp. That is to say training 3D detection with depth is important.
Is my understanding right？

**Comment by chenyilun95** [comment]
Yes, they are same thing. It's important due to its performance gain.
